Makefile

```diff
@@ -84,6 +85,10 @@ docstrings:
 checkdocstrings:
 	tox -e py36-checkdocstrings
 
+.PHONY: pip-compile
+pip-compile:
+	tox -qe py36-dev -- pip-compile --output-file requirements.txt requirements.in
```

Comment:
The current version of `requirements.txt` is compiled for a Python 2.7 environment. Compiling it under Python 3.6 results in some conditional dependencies being removed (eg functools32).

Reply:
Fixed